### ISSUE TITLE
docs: SSH not restarting with "sudo systemctl restart sshd.service"

### ISSUE THREAD

**Issue by WJ4IoT** [canonical/ubuntu-server-documentation#93]
I notice the command "sudo systemctl restart sshd.service"" does not work on my system:
and "Failed to restart sshd.service: Unit sshd.service not found."

where the command "sudo systemctl restart ssh.service" just works.

I do not have an original "/etc/ssh/sshd_config" anymore because i am working on an older system but this system is now updated to the latest Ubuntu LTS server version (Ubuntu 24.04.1 LTS (GNU/Linux 6.8.0-50-generic x86_64), I do have "/etc/ssh/sshd_config.ucf-dist"),

How do I become in this case a fresh copy of "/etc/ssh/sshd_config"?
---
*Reported from: https://documentation.ubuntu.com/server/how-to/security/openssh-server/*

**Comment by cpaelzer** [comment]
Hi @WJ4IoT ,
I first wanted to say that seems like a config issue, but it might be legit the more I look into it.

I noticed it was good earlier (e.g. jammy) and later (e.g. plucky) in a quick re-check.
While looking if I should file that bug I spotted that I'm about a month too late => https://bugs.launchpad.net/ubuntu/+source/openssh/+bug/2087949

It is already being fixed (that is why it works in plucky) and on the way for the fix to be backported.

For the docs, we could just make it `sshd`->`ssh` to avoid that issue immediately. And while `ssh.service` isn't as clear to be "the daemon" it is the primary name of the service file - so that should be fine.
